### DISCUSSION TITLE
Power on animation for cake

### DISCUSSION THREAD

**Discussion by Toy203** [gpstar81/GPStar-proton-pack#327] [Ideas]
Would be cool to extend the pack power on animation to the cake leds if the cyclotron lid is off (or by default anyway). Just a full "rotation" should fit the sound timing well.

**Comment by nomakewan** [comment]
Does the inner cyclotron not already slowly ramp up on start? Or are you looking for something else?

**Comment by Toy203** [comment]
I meant for the Power On animation when you plug in the battery. Currently it doesn't have any animation.

**Comment by nomakewan** [comment]
Ah the Power On Self Test! Okay, yeah, that makes sense. I'll see if I can throw something together for 5.3.4 then.

**Comment by Toy203** [comment]
Glad to hear that is the official name. I even added a label.

**Reply by Toy203** [reply]
![1000061507](https://github.com/user-attachments/assets/878ebce1-17c8-4198-be94-3fc5f781bc1b)
